style.css

```diff
@@ -19,6 +19,32 @@ body {
   box-sizing: border-box;
 }
 
+.hidden {
+  display: none;
+}
+
+/* menu */
+.navbar {
+  transition: background-color 0.3s, color 0.3s;
+}
+
+.bg-transparent {
+  background-color: transparent !important;
+  color: white !important;
+}
+
+.text-white {
+  color: white !important;
+}
+
+.bg-light {
+  background-color: #fff !important;
+}
+
+.text-dark {
+  color: #333 !important;
```

Comment:
musisz wszedzie tego important uzywac? Raczej staramy sie tego slowka nie uzywac bo zazwyczaj to znaczy ze nie jest wystarczajaco dobrze zrobiony css. bo important wymusza inne zachowanie. A raczej wystarczy nadpisac kolor np i tyyle